src/vmshepherd_runtime_postgres_driver/__init__.py

```diff
@@ -1,9 +1,33 @@
 from datetime import datetime
 import json
+import logging
 import asyncpg
 from vmshepherd.runtime import AbstractRuntimeData
 
 
+queries = {
```

Comment:
I thought we discouraged such a form (queries separated from a place of use)

Reply:
I agree with you, back to inline queries.